File: modul5/Python/PRAK504-2310817210023-RifkyPutraMahardika.py
def reverse(angka):
    Reverseangka = 0
    while angka != 0:
        digit = angka % 10
        Reverseangka = Reverseangka * 10 + digit
        angka //= 10
    return Reverseangka

A, B = map(int, input().split())
A = reverse(A)
B = reverse(B)
C = A + B
print(reverse(C))

File: modul5/C/PRAK504-2310817210023-RifkyPutraMahardika.c
#include <stdio.h>
int reverse(int angka) {
    int Reverseangka = 0;
    while (angka != 0) {
        int digit = angka % 10;
        Reverseangka = Reverseangka * 10 + digit;
        angka /= 10;
    }
    return Reverseangka;
}
int main() {
 int A, B;
 scanf("%d %d",&A,&B);
 A=reverse(A);
 B=reverse(B);
 int C = A+B;
 printf("%d",reverse(C));
}